gridstatus/ercot_api/ercot_api.py

```diff
@@ -1191,6 +1196,67 @@ def get_60_day_dam_disclosure(
         # Take the list of dictionaries and concat the dataframes for each key
         return {key: pd.concat([d[key] for d in df_list]) for key in df_list[0].keys()}
 
+    @support_date_range(frequency=None)
+    def get_60_day_sced_disclosure(
```

Comment:
would it be straightforward to make `process` a parameter here like we do for `Ercot.get_60_day_sced_disclosure`?